cylc/flow/task_job_mgr.py

```diff
@@ -839,7 +887,12 @@ def _submit_task_job_callback(self, suite, itask, cmd_ctx, line):
             if ctx.ret_code:
                 ctx.cmd = cmd_ctx.cmd  # print original command on failure
         log_task_job_activity(ctx, suite, itask.point, itask.tdef.name)
-
+        if cmd_ctx.ret_code == 255:
+            # Submission failed because host was unreachable. Retry NOW.
+            self.task_events_mgr._retry_task(
```

Comment:
`_retry_task` seems like a rather low level psuedo-private function, but by using it I bypass using the usual submit fail/retry retry logic which I'm not sure that I want to touch.